app/_src/kubernetes-ingress-controller/guides/redis-rate-limiting.md

```diff
@@ -307,13 +276,12 @@ The results should look like this:
 ```
 {% endif_version %}
 
-## Test it
+## Test rate limiting is a multi-node Kong deployment
```

Comment:
I'd move this section above the optional Secrets Management section

Reply:
Done!